.github/workflows/coverage_comment.yml

```diff
@@ -0,0 +1,79 @@
+name: Coverage Comment
+
+on:
+  workflow_run:
+    workflows: ["Coverage Tests"]
+    types:
+      - completed
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  comment:
+    if: >
+      github.event.workflow_run.event == 'pull_request' &&
+      github.event.workflow_run.conclusion == 'success' &&
+      github.repository == 'django/django'
+    name: Post Coverage Comment
+    runs-on: windows-latest
+    steps:
+      - name: Download diff coverage report
+        uses: actions/download-artifact@v4
+        with:
+          name: diff-coverage-report-${{ github.event.workflow_run.pull_requests[0].number }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          run-id: ${{ github.event.workflow_run.id }}
+
+      - name: Download PR number
+        uses: actions/download-artifact@v4
+        with:
+          name: pr-number-${{ github.event.workflow_run.pull_requests[0].number }}
```

Comment:
If we're extracting the pr number here, do we need to have a pr_number.txt artifact?

Reply:
hmm, we should be able to use that directly.
checking locally.